docset/winserver2022-ps/addsdeployment/Test-ADDSDomainControllerUninstallation.md

```diff
@@ -11,52 +11,69 @@ title: Test-ADDSDomainControllerUninstallation
 # Test-ADDSDomainControllerUninstallation
 
 ## SYNOPSIS
+
 Runs the prerequisites for uninstalling a domain controller in Active Directory.
 
 ## SYNTAX
 
 ### ADDSDomainControllerUninstall (Default)
+
 ```
 Test-ADDSDomainControllerUninstallation [-LocalAdministratorPassword <SecureString>]
- [-Credential <PSCredential>] [-DemoteOperationMasterRole] [-DnsDelegationRemovalCredential <PSCredential>]
- [-IgnoreLastDCInDomainMismatch] [-IgnoreLastDnsServerForZone] [-LastDomainControllerInDomain]
- [-NoRebootOnCompletion] [-RemoveApplicationPartitions] [-RemoveDnsDelegation] [-RetainDCMetadata] [-Force]
+ [-Credential <PSCredential>] [-DemoteOperationMasterRole]
+ [-DnsDelegationRemovalCredential <PSCredential>] [-IgnoreLastDCInDomainMismatch]
+ [-IgnoreLastDnsServerForZone] [-LastDomainControllerInDomain] [-NoRebootOnCompletion]
+ [-RemoveApplicationPartitions] [-RemoveDnsDelegation] [-RetainDCMetadata] [-Force]
  [<CommonParameters>]
 ```
 
 ### ADDSDomainControllerUninstallForceRemoval
+
 ```
 Test-ADDSDomainControllerUninstallation [-LocalAdministratorPassword <SecureString>]
- [-Credential <PSCredential>] [-DemoteOperationMasterRole] [-ForceRemoval] [-NoRebootOnCompletion] [-Force]
- [<CommonParameters>]
+ [-Credential <PSCredential>] [-DemoteOperationMasterRole] [-ForceRemoval]
+ [-NoRebootOnCompletion] [-Force] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Test-ADDSDomainControllerUninstallation** cmdlet runs those prerequisite checks which would be performed if you were to use the Uninstall-ADDSDomainController cmdlet to uninstall a domain controller in Active Directory.
-It differs from using the *WhatIf* parameter with the **Uninstall-ADDSDomainController** cmdlet in that instead of summarizing the changes that would occur during the uninstallation process, this cmdlet actually tests whether those changes are possible given the current environment.
 
-For more information on the scope of these prerequisite checks that the ADDSDeployment module performs when using this cmdlet see the section ADPrep and Prerequisite Checking Architecture in [AD DS Simplified Administration](https://go.microsoft.com/fwlink/?LinkID=237244).
+The `Test-ADDSDomainControllerUninstallation` cmdlet runs those prerequisite checks which would be
+performed if you were to use the Uninstall-ADDSDomainController cmdlet to uninstall a domain
+controller in Active Directory. It differs from using the `-WhatIf` parameter with the
+`Uninstall-ADDSDomainController` cmdlet in that instead of summarizing the changes that would occur
+during the uninstallation process, this cmdlet actually tests whether those changes are possible
+given the current environment.
+
+For more information on the scope of these prerequisite checks that the ADDSDeployment module
+performs when using this cmdlet see the section ADPrep and Prerequisite Checking Architecture in
+[AD DS Simplified Administration](https://go.microsoft.com/fwlink/?LinkID=237244).
```

Comment:
```suggestion
The `Test-ADDSDomainControllerUninstallation` cmdlet runs those prerequisite checks which would be
performed if you were to use the `Uninstall-ADDSDomainController` cmdlet to uninstall a domain
controller in Active Directory. It differs from using the **WhatIf** parameter with the
`Uninstall-ADDSDomainController` cmdlet in that instead of summarizing the changes that would occur
during the uninstallation process, this cmdlet actually tests whether those changes are possible
given the current environment.

For more information on the scope of these prerequisite checks that the **ADDSDeployment** module
performs when using this cmdlet see the section ADPrep and Prerequisite Checking Architecture in
[AD DS Simplified Administration](https://go.microsoft.com/fwlink/?LinkID=237244).
```